packages/jupyter-ai/src/components/chat.tsx

```diff
@@ -202,51 +207,57 @@ export function Chat(props: ChatProps): JSX.Element {
     <JlThemeProvider themeManager={props.themeManager}>
       <SelectionContextProvider selectionWatcher={props.selectionWatcher}>
         <CollaboratorsContextProvider globalAwareness={props.globalAwareness}>
-          <Box
-            // root box should not include padding as it offsets the vertical
-            // scrollbar to the left
-            sx={{
-              width: '100%',
-              height: '100%',
-              boxSizing: 'border-box',
-              background: 'var(--jp-layout-color0)',
-              display: 'flex',
-              flexDirection: 'column'
-            }}
+          <ActiveCellContextProvider
+            activeCellManager={props.activeCellManager}
           >
-            {/* top bar */}
-            <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
-              {view !== ChatView.Chat ? (
-                <IconButton onClick={() => setView(ChatView.Chat)}>
-                  <ArrowBackIcon />
-                </IconButton>
-              ) : (
-                <Box />
+            <Box
+              // root box should not include padding as it offsets the vertical
+              // scrollbar to the left
+              sx={{
+                width: '100%',
+                height: '100%',
+                boxSizing: 'border-box',
+                background: 'var(--jp-layout-color0)',
+                display: 'flex',
+                flexDirection: 'column'
+              }}
+            >
+              {/* top bar */}
+              <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+                {view !== ChatView.Chat ? (
+                  <IconButton onClick={() => setView(ChatView.Chat)}>
+                    <ArrowBackIcon />
+                  </IconButton>
+                ) : (
+                  <Box />
+                )}
+                {view === ChatView.Chat ? (
+                  <IconButton onClick={() => setView(ChatView.Settings)}>
+                    <SettingsIcon />
+                  </IconButton>
+                ) : (
+                  <Box />
+                )}
```

Comment:
These two expressions test, effectively, the same condition. Could they be collapsed to one ternary expression?